### ISSUE TITLE
v0.3.0 and above break Netlify builds

### ISSUE THREAD

**Issue by cspetterson** [mhanberg/jekyll-postcss#15]
As you state in your CHANGELOG, there is a breaking change in v0.3.0

This breaks Netlify builds.

The builds on Netlify seem to hang with no information, but I figured I would share that problem here since this is the cause. I managed to get my build deployed successful by downgrading the gem to the previous version (0.2.2) - but this is a shame, since the latest version is 0.4.0

More information can be found on the Netlify thread here: https://community.netlify.com/t/deploy-is-hanging-postcss-problem/14822/7

**Comment by mhanberg** [comment]
Hi,

I did not get any notifications about that thread (as you have probably noticed, I am the one who opened it), so I had not idea that they replied.

I have already implemented a workaround for this, which is to not start the server when running outside of development mode, and this was released in [v0.3.1](https://github.com/mhanberg/jekyll-postcss/blob/master/CHANGELOG.md#031) two weeks ago.

The change you are referencing in the v0.3.0 release notes is unrelated to the hanging Netlify builds.
